e2e_test/js-tests/send_tx.mjs

```diff
@@ -59,7 +59,7 @@ async function main() {
     value: value,
     gas: 90000,
     feeCurrency,
-    maxFeePerGas: 25000000000n,
+    maxFeePerGas: 50000000002n,
```

Comment:
`25000000000` was considerably higher than the base fee floor, so the transaction was included in a block. Now calculate the gas price with the base fee floor in mind and set it only fractionally above that threshold.

Reply:
@piersy because this transaction won't be paid in CELO.

Reply:
The previous `maxFeePerGas` value (`25000000000`) was 100 times of the previous base-fee floor, so both legacy and fee-currency transactions were accepted with that `maxFeePerGas`.
Now, current maxFeePerGas `50000000002` should be the minimum acceptable price for fee currency transaction.

Reply:
Ok got you, thanks!